### PR TITLE
fix broken links

### PR DESCRIPTION
**Pull request by mairaw**

### CODE REVIEW

File: docs/standard/microservices-architecture/secure-net-microservices-web-applications/index.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ In microservice scenarios, authentication is typically handled centrally. If you
 
 **Figure 9-1**. Centralized authentication with an API Gateway
 
-If services can be accessed directly, an authentication service like Azure Active Directory or a dedicated authentication microservice acting as a security token service (STS) can be used to authenticate users. Trust decisions are shared between services with security tokens or cookies. (These tokens can be shared between applications, if needed, in ASP.NET Core with [data protection services](/aspnet/core/security/data-protection/compatibility/cookie-sharing#sharing-authentication-cookies-between-applications).) This pattern is illustrated in Figure 9-2.
+If services can be accessed directly, an authentication service like Azure Active Directory or a dedicated authentication microservice acting as a security token service (STS) can be used to authenticate users. Trust decisions are shared between services with security tokens or cookies. (These tokens can be shared between applications, if needed, in ASP.NET Core with [data protection services](/aspnet/core/security/cookie-sharing).) This pattern is illustrated in Figure 9-2.
```
Comment (mvelosop) [inline docs/standard/microservices-architecture/secure-net-microservices-web-applications/index.md]:
```suggestion
If services can be accessed directly, an authentication service like Azure Active Directory or a dedicated authentication microservice acting as a security token service (STS) can be used to authenticate users. Trust decisions are shared between services with security tokens or cookies. (These tokens can be shared between by ASP.NET Core applications, if needed, by [sharing cookies](/aspnet/core/security/cookie-sharing).) This pattern is illustrated in Figure 9-2.
```

Comment (mvelosop) [inline docs/standard/microservices-architecture/secure-net-microservices-web-applications/index.md]: @mairaw the link correction is Ok, I think perhaps the redaction should be tweaked a little, something like this:

```suggestion
If services can be accessed directly, an authentication service like Azure Active Directory or a dedicated authentication microservice acting as a security token service (STS) can be used to authenticate users. Trust decisions are shared between services with security tokens or cookies. (These tokens can be shared between ASP.NET Core applications, if needed, by [cookie sharing](/aspnet/core/security/cookie-sharing).) This pattern is illustrated in Figure 9-2.
```

Comment (mvelosop) [inline docs/standard/microservices-architecture/secure-net-microservices-web-applications/index.md]: @mairaw the link correction is Ok, I think perhaps the redaction should be tweaked a little, something like this:

```suggestion
If services can be accessed directly, an authentication service like Azure Active Directory or a dedicated authentication microservice acting as a security token service (STS) can be used to authenticate users. Trust decisions are shared between services with security tokens or cookies. (These tokens can be shared between ASP.NET Core applications, if needed, by implementing [cookie sharing](/aspnet/core/security/cookie-sharing).) This pattern is illustrated in Figure 9-2.
```

File: docs/standard/microservices-architecture/secure-net-microservices-web-applications/index.md [hunk 2]
```diff
@@ -271,7 +271,7 @@ The JWT bearer authentication middleware can also support more advanced scenario
 ## Additional resources
 
 - **Sharing cookies between applications** \
-  [*https://docs.microsoft.com/aspnet/core/security/data-protection/compatibility/cookie-sharing\#sharing-authentication-cookies-between-applications*](/aspnet/core/security/data-protection/compatibility/cookie-sharing#sharing-authentication-cookies-between-applications)
+  [*https://docs.microsoft.com/aspnet/core/security/cookie-sharing\*](/aspnet/core/security/cookie-sharing)
```
Comment (mvelosop) [inline docs/standard/microservices-architecture/secure-net-microservices-web-applications/index.md]: The backslash looks odd:

```suggestion
  [*https://docs.microsoft.com/aspnet/core/security/cookie-sharing/*](/aspnet/core/security/cookie-sharing)
```

**Review by mvelosop — CHANGES_REQUESTED** [review]

**Comment by mvelosop** [comment]
@mairaw, I just finished my review, but I'm not sure I did it the right way, would you confirm, please? 😊

**Comment by mairaw** [comment]
Yes you @mvelosop. I've applied your suggestions + another small fix

**Review by mvelosop — APPROVED** [review]

**Pull request merged**